environments/perl/server.pl

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env perl
+
+use utf8;
+use strict;
+use warnings;
+
+use Getopt::Args;
+use Plack::Handler::Twiggy;
+
+opt codepath => (
+    isa     => 'Str',
+    default => '/userfunc/user',
+    comment => 'Path to the user code',
+);
+
+opt port => (
+    isa     => 'Int',
+    default => 8888,
+    comment => 'Port to listen on',
+);
+
+my $options = optargs();
+
+{
+    package App::Fission::Perl;
+
+    use Dancer2;
+
+    our $userfunc;
+
+    post '/specialize' => sub {
+        if($userfunc) {
+            send_error('Already done', 500);
+        }
+
+        $userfunc = require($options->{codepath});
+        return '';
+    };
+
+    get '/' => sub {
```

Comment:
Is it possible to make this endpoint respond to any http method? Such that you could use GET, POST, DELETE etc. to make it consistent with other environments.